src/main/java/com/scalar/cassy/traverser/FileTraverser.java

```diff
@@ -39,7 +39,7 @@ public FileTraverser(Path dataDir) {
           }
         });
 
-    return filePaths.stream().map(f -> dataDir.relativize(f)).collect(Collectors.toList());
+    return filePaths;
```

Comment:
Now, Traverser returns full path. Getting relative path is now done in Uploader.